src/hal/src/pso/mod.rs

```diff
@@ -151,26 +151,52 @@ impl fmt::Display for Stage {
 }
 
 /// Shader entry point.
-#[derive(Debug, Copy)]
-pub struct EntryPoint<'a, B: Backend> {
+#[derive(Debug)]
+pub struct EntryPoint<'a, B: Backend, S = RawSpecialization<'a>> {
     /// Entry point name.
     pub entry: &'a str,
     /// Shader module reference.
     pub module: &'a B::ShaderModule,
     /// Specialization.
-    pub specialization: Specialization<'a>,
+    pub specialization: S,
+}
+
+impl<'a, B, S> EntryPoint<'a, B, S>
+where
+    B: Backend,
+    S: Specialization<'a>,
+{
+    /// Convert `specialization` of the `EntryPoint` into raw type
+    /// acceptable by pipeline creation functions.
+    pub fn into_raw(&self, storage: &'a mut SpecializationStorage) -> EntryPoint<'a, B> {
```

Comment:
this method should follow rust conventions. `into_` is supposed to be moving out of `self`

Reply:
Okey